apps/playground/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "playground",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "start:storybook": "vite preview",
+    "storybook": "VITE_CJS_IGNORE_WARNING=true storybook dev -p 6006",
```

Comment:
Is the `VITE_CJS_IGNORE_WARNING=true` required here? Otherwise, wouldn't we want to see all the warnings we could to effectively remove them?

Reply:
It's not required it's just to suppress some warnings. So technically we could remove it and tackle the warning (or ignore it)

Reply:
Removed it with lastest commit